exercises/flower-field/introduction.md

```diff
@@ -0,0 +1,7 @@
+# Introduction
+
+[Flower Field][Wikipedia] is a compassionate reimagining of the popular game Minesweeper.
```

Comment:
If we are doing away with triggers, then we should really not mention the honored occupation, as well.

Reply:
```suggestion
[Flower Field (trigger warning)][history] is a compassionate reimagining of a popular game.
```

and then

```
[history]: https://web.archive.org/web/20020409051321fw_/http://rcm.usr.dsi.unimi.it/rcmweb/fnm/
```

No need to link to wiki ;)

Reply:
Also, hopefully it keeps the issues alive, as there are still a lot of mines out there that need to be swept and we get more people trained to care for this problem.  So not unnecessarily exposing those traumatized by it should not be enough to throw the problem under the rug where no one sees that it is something that still needs to be addressed.

Reply:
> Also, hopefully it keeps the issues alive, as there are still a lot of mines out there that need to be swept and we get more people trained to care for this problem.

... and we get more people _and [rats](https://en.wikipedia.org/wiki/Demining#Rats
)_ trained to care for this problem.

[APOPO](https://en.wikipedia.org/wiki/APOPO) credits the rats with clearing more than 100,000 mines.